src/core/Jalasoft.TeamUp.Projects.Core/ProjectsService.cs

```diff
@@ -25,14 +25,14 @@ public Project GetProject(Guid id)
                 var project = this.projectsRepository.GetById(id);
                 if (project == null)
                 {
-                    throw new ProjectsException(ProjectsException.ProjectsErros.NotFound);
+                    throw new ProjectsException(ProjectsException.ProjectsErrors.NotFound);
                 }
 
                 return project;
             }
             catch (Exception)
             {
-                throw new ProjectsException(ProjectsException.ProjectsErros.InternalServerError);
+                throw new ProjectsException(ProjectsException.ProjectsErrors.InternalServerError);
```

Comment:
the same here, if an exception raises in the service layer it should be handled at API level
btw here we are missing the stack trace, please we need to have the stack trace to log it later otherwise we'll be blind when an error occurs
https://www.thecodebuzz.com/understanding-throw-keyword-in-exception-handling/

Reply:
done